packages/client/tests/e2e/nextjs-schema-not-found/10_monorepo-serverComponents-customOutput-noReExport/_steps.ts

```diff
@@ -8,7 +8,6 @@ void executeSteps({
     await $`pnpm install`
     cd('packages/service')
     await $`pnpm exec prisma db push --force-reset`
-    await $`pnpm exec next build`
```

Comment:
Why could all these be removed?

Reply:
Found it in `packages/client/tests/e2e/nextjs-schema-not-found/_shared/test.ts` 👍